code/restful-api/restful-api/Models/Historico_Eventos.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace RestfulApi.Models
+{
+    public class Historico_Eventos
```

Comment:
I think we don't need this class, since it's just going to be a consult on the DB like 'give me all events that this user with id X came'. What do you think?

Reply:
Retirado.